doc/source/notebooks/advanced/kernels.ipynb

```diff
@@ -403,7 +403,7 @@
    "source": [
     "## Specify active dimensions\n",
     "\n",
-    "When combining kernels, it's often helpful to have bits of the kernel working on different dimensions. For example, to model a function that is linear in the first dimension and smooth in the second, we could use a combination of Linear and Matern52 kernels, one for each dimension.\n",
+    "When combining kernels, it's often helpful to have bits of the kernel working on different dimensions. For example, to model a function that is linear in the first dimension and smooth in the second, we could use a combination of linear and Matern52 kernels, one for each dimension.\n",
```

Comment:
Should this not be "Linear" like the kernel class ? Alternatively, if we want to refer to the mathematical construct, it should also be Matérn 5/2 kernel instead of Matern52...